packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -155,6 +158,7 @@ export const getMaxMinSell = (
   orderType: SBOrderActionType,
   quote: { quote: SwapQuoteType; rate: number },
   pair: SBPairType,
+  payment: undefined | PaymentValue,
```

Comment:
do we need explicitly to define `undefined`? Can it just be `payment?: PaymentValue,`?

Reply:
same question for line 79